Specs/DataSources/KmlDataSourceSpec.js

```diff
@@ -323,6 +324,21 @@ defineSuite([
         });
     });
 
+    it('raises unsupportedNodeEvent event when parsing an unsupported kml node type', function() {
```

Comment:
This test fails. The event is raised with arguments of the datasource and the node, but this test is checking for a datasource and a boolean. You are also checking for the spy to have been called before the promise resolves, but that won't happen. The test should probably be something like this to test that both arguments are correct.

```
        var dataSource = new KmlDataSource(options);
        var spy = jasmine.createSpy('unsupportedNodeEvent');
        dataSource.unsupportedNodeEvent.addEventListener(spy);

        return dataSource.load('Data/KML/unsupported.kml').then(function() {
            var nodeNames = ['GroundOverlay', 'PhotoOverlay', 'ScreenOverlay', 'Tour'];
            expect(spy.calls.count()).toEqual(4);
            for (var i = 0; i < nodeNames.length; ++i) {
                var args = spy.calls.argsFor(i);
                expect(args.length).toEqual(2);
                expect(args[0]).toBe(dataSource);
                expect(args[1].localName).toEqual(nodeNames[i]);
            }
        });
```

I haven't tested this, so please run the tests and make sure they pass. Thanks.